File: applets/src/org/rr0/ufoathome/model/sky/SkyEvent.java
package org.rr0.ufoathome.model.sky;

/**
 * @author Jerôme Beau
 * @version 4 déc. 2003 23:08:42
 */
public class SkyEvent extends java.util.EventObject {
    protected Object value;

    public SkyEvent(Object source, Object value) {
        super(source);
        this.value = value;
    }

    public Object getValue() {
        return value;
    }
}

File: applets/src/org/rr0/ufoathome/model/sky/SkyEvent.ts
import {EventObject} from "../../view/gui/EventObject";

export class SkyEvent extends EventObject {
  protected value: Object;

  constructor(source: Object, value: Object) {
    super(source);
    this.value = value;
  }

  public getValue(): Object {
    return this.value;
  }
}
